### DISCUSSION TITLE
Is there a how to or blueprint for backup Notifications?

### DISCUSSION THREAD

**Discussion by Stephan-4711** [lavinir/hassio-onedrive-backup#176] [Ideas]
Is there a blueprint or a how to how to create notification automations for a backup report? For example success full backup, upload status ok, storage left on HA, left on onedrive. Currently X backups are stored online/local
And so on

**Comment by DennisAppelman** [comment]
i was looking for this as well..

**Comment by lavinir** [comment]
I don't know if anyone has created any blueprints but you can make automations for most of the things you listed using the exposed sensors, all detailed in the readme.
See: 
https://github.com/lavinir/hassio-onedrive-backup/tree/main?tab=readme-ov-file#onedrive-free-space-sensor
https://github.com/lavinir/hassio-onedrive-backup/tree/main?tab=readme-ov-file#home-assistant-backup-sensor
